packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -71,6 +111,11 @@ If all prerequisites in the previous section are met, enabling the feature can b
 
 In case of emergency grouped batching can be disabled at runtime, using feature gates. If `"Fluid.ContainerRuntime.DisableGroupedBatching"` is set to `true`, it will disable grouped batching if enabled from `IContainerRuntimeOptions` in the code.
 
+### Notes
```

Comment:
I'd probably not have that line (isolate into sub-section) - sections are small, and "Notes" are not in the list of content, so value of that is low. Same for next section